docs/conf.py

```diff
@@ -0,0 +1,212 @@
+import sys
+import os
+import requests
+from urllib.parse import urlparse
+from git import Repo
+import subprocess
+import time
+
+sys.path.append('./')
+from custom_conf import *
+sys.path.append('.sphinx/')
+from build_requirements import *
+
+# Configuration file for the Sphinx documentation builder.
+# You should not do any modifications to this file. Put your custom
+# configuration into the custom_conf.py file.
+# If you need to change this file, contribute the changes upstream.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
```

Comment:
From experience with other projects, most of the sphinx config can be left at defaults. This file, in the current state, is probably too long or too verbose.

It's a bit of work though to go through every setting to validate if it's needed and what it really does.

Reply:
I think this file is customized by the starter pack and that's why they keep it in the pack. The starter pack probably won't work if we remove the whole thing and use default config only.